### PR TITLE
Continuation of #286

### PR DESCRIPTION
**Pull request by hiromis**
See #286 for details.

This PR was created since the original working branch for #286 is now a protected branch and PR comments cannot be addressed in that branch. A new branch was created off of release-2.0 and merged the original commit as well as the changes initiated by the PR comment.

### CODE REVIEW

File: greymatter.yaml [hunk 1]
```diff
@@ -17,7 +17,8 @@ global:
     version: 1.0.1
     sidecar:
       version: 0.9.1
-
+  sense:
+    enableed: false
```
Comment (tilleryd) [inline greymatter.yaml]: `enabled`

File: dashboard/templates/prometheus-configmap.yaml [hunk 2]
```diff
@@ -36,6 +36,26 @@ data:
...
+        - source_labels: ['__meta_kubernetes_pod_container_port_name']
+          regex: 'metrics'
+          action: 'keep'
+        # Relabel Jobs to the service name and version of the zk path
+        - source_labels: ['__meta_kubernetes_pod_label_app']
+          regex: '(.*)'
+          target_label:  'job'
+          #replacement:   '${1}'
+          replacement:   '${1}'
+      {{- end }}
```
Comment (tilleryd) [inline dashboard/templates/prometheus-configmap.yaml]: There should also be a corresponding config in `dashboard/values.yaml` representing the default if there's no sense config in the global values file.

```
sense:
  enabled: false
```

Comment (hiromis) [reply dashboard/templates/prometheus-configmap.yaml]: 👍 Done!

File: greymatter.yaml (hunk 1 above)
Comment (hiromis) [reply greymatter.yaml]: Fixed.

**Comment by hiromis** [comment]
Merge conflicts resolved and re-requested reviews.

**Pull request merged**